python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -80,6 +83,11 @@ def _create_channel_ref(
     return object_ref
 
 
+class ResizeChannel:
```

Comment:
Add docstring.

Also, we should rename to `_ResizeChannel` since this class is only used internally.

Reply:
Done.